libminifi/src/core/extension/ExtensionManager.cpp

```diff
@@ -72,9 +72,13 @@ bool ExtensionManager::initialize(const std::shared_ptr<Configure>& config) {
     }));
     for (const auto& candidate : candidates) {
       auto library = internal::asDynamicLibrary(candidate);
-      if (!library || !library->verify(logger_)) {
+      if (!library) {
         continue;
       }
+      if (!library->verify(logger_)) {
+        logger_->log_warn("Skipping library '{}' at '{}': failed verification, different build?",
+            library->name, library->getFullPath());
+      }
```

Comment:
we log that we are skipping it, but we aren't: is a `continue` missing?

Reply:
Yup, that was an oversight, thanks for catching it